sql/2024/accessibility/a11y_technology_usage.sql

```diff
@@ -0,0 +1,18 @@
+#standardSQL
```

Comment:
@tunetheweb If I want to replace this code, and use a is\_root\_page column, I just do this, right:

```
#standardSQL
# Accessibility (A11y) technology, ie. Overlays, usage by client


SELECT
client, # Client domain
is_root_page,
rank ,
COUNT(DISTINCT page) AS total_sites, # Total number of unique sites for the client
COUNT(DISTINCT IF(category = 'Accessibility', page, NULL)) AS sites_with_a11y_tech, # Number of unique sites that use accessibility technology
COUNT(DISTINCT IF(category = 'Accessibility', page, NULL)) / COUNT(DISTINCT page) AS pct_sites_with_a11y_tech # Percentage of sites using accessibility technology
FROM
`httparchive.all.pages`,
UNNEST(technologies) AS tech,
UNNEST(categories) AS category
WHERE
date = '2024-06-01' # Specific date for data extraction
GROUP BY
client, # Group by client domain
is_root_page
ORDER BY
client; # Order results by client domain
```

Reply:
LGTM! But I'd also add `is_root_page` to your `ORDER BY` clause so you always get those in a consistent order.

Reply:
Looks like I need to add it to `GROUP BY` as well.